detection-rules/impersonation_vanta.yml

```
@@ -0,0 +1,24 @@
+name: "Brand impersonation: Vanta"
+description: |
+  Impersonation of Vanta.
+type: "rule"
+severity: "low"
+authors: |
```

```suggestion
authors:
```

`|` is for multi-line strings